spacy_llm/tasks/textcat.py

```diff
@@ -137,10 +137,13 @@ def _format_response(self, response: str) -> Dict[str, float]:
         The returned dictionary contains the labels mapped to their score.
         """
         categories: Dict[str, float]
+        response = self._normalizer(response.strip())
```

Comment:
Should we call `response.strip()` here or rely on the normalizer to strip. 

I can't really think of a scenario where we want whitespace at the beginning or end of the response here if a user wanted to provide their own normalizer but I suppose it's possible.

Reply:
I'm leaning towards delegating/relying it to the normalizer so that it's not "hidden" 🤔 and perhaps also for consistency (not too attached though, I agree we'd always want to strip())!